### PR TITLE
Add dropdown with creating project button for "New issue" button in tracker

### PR DESCRIPTION
**Pull request by ThetaDR**
# Contribution checklist

## Brief description

image

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [x] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [x] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: plugins/tracker-resources/src/components/NewIssueHeader.svelte [hunk 1]
```diff
@@ -39,27 +40,42 @@
       closed = true
     })
   }
+  const dropdownItems: SelectPopupValueType[] = [
+    {
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/NewIssueHeader.svelte:45]: We need both CreateIssue and CreateProject in dropdown I suppose.

Comment (ThetaDR) [reply plugins/tracker-resources/src/components/NewIssueHeader.svelte:45]: I disagree. I don't think we need to double the same action when we already have it right there

File: plugins/tracker-resources/src/components/NewIssueHeader.svelte [hunk 2]
```diff
@@ -15,11 +15,12 @@
 <script lang="ts">
   import { Ref, Space } from '@hcengineering/core'
   import { MultipleDraftController, getClient } from '@hcengineering/presentation'
-  import { Button, IconAdd, showPopup } from '@hcengineering/ui'
+  import { IconAdd, IconDropdown, SelectPopupValueType, showPopup } from '@hcengineering/ui'
   import view from '@hcengineering/view'
   import { onDestroy } from 'svelte'
   import tracker from '../plugin'
   import CreateIssue from './CreateIssue.svelte'
+  import ButtonWithDropdown from '@hcengineering/ui/src/components/ButtonWithDropdown.svelte'
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/NewIssueHeader.svelte]: Please use proper import

File: plugins/tracker-resources/src/components/NewIssueHeader.svelte [hunk 3]
```diff
@@ -39,27 +40,52 @@
...
   }
+  const dropdownItems: SelectPopupValueType[] = [
+    {
+      id: tracker.string.CreateProject,
+      label: tracker.string.CreateProject
+    },
+    {
+      id: tracker.string.NewIssue,
+      label: tracker.string.NewIssue
+    }
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/NewIssueHeader.svelte:52]: We need resume draft if we in draft mode, and new issue here probable should not respect draft.

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**